default_launch.json

```diff
@@ -16,26 +16,35 @@
       }
     ]
   },
+  "load_balancing": {
+    "layer_configurations": [
+      {
+        "layer": "UnityGameLogic",
+        "hex_grid": {
+          "num_workers": 2
```

Comment:
This is a different configuration to the previous JSON

Reply:
there's no direct equivalent for the `singleton` strategy used before

I could switch this to POI with one point at the origin?